### ISSUE TITLE
find mode with the selected text

### ISSUE THREAD

**Issue by whwnow** [philc/vimium#3325]
set the default value with selected text in find mode，no need ctrl +v

**Comment by innaterebel** [comment]
Sounds like you are frequently reusing a certain keyword. Find mode actually has a history record, accessible via <kbd>↑</kbd><kbd>↓</kbd> keys within find mode.
This history is sorted by recently used order, so you can quickly reuse the same word pressing <kbd>/</kbd>then <kbd>↑</kbd>.

**Comment by whwnow** [comment]
Apologize for my bad English.
What I want to express is to query the other text of the current page through the text I selected. Not the query history

**Comment by innaterebel** [comment]
Ah, it was me misunderstood your point, sorry.
What is your use case for this? Do you mean:

1. After selecting text via visual mode, while still in visual mode (e.g. has not yanked them yet), `/` will start find mode directly with the text selected.
2. Has the current copied text automatically pasted into find mode when invoked.

Although it has to be decided by the committers, Vimium has the [design goals](https://github.com/philc/vimium/blob/master/CONTRIBUTING.md#what-makes-for-a-good-feature-requestcontribution-to-vimium) to only implement features/options that will be useful to many users.
If it is 2., I imagine more users are likely to be confused when they find their randomly copied text sitting in find mode when invoke it.

**Comment by whwnow** [comment]
The first point is the function I want to express.  Is it a feature that might be added?
My current operation:   `select text` → `ctrl + c` → `/` → `ctrl + v`
I think it is better: `select text` → `/`

**Comment by XinAnzzZ** [comment]
When i want open history or bookmark entry in the new tab, I press "O" key and i type "xxx" , the top result in the result list is what i want, but i must press `↓` and `enter`.

So I hope add a key map to open the top result directly. For example ` cmd + enter`. 

Thanks!

**Comment by scutxd** [comment]
I don't think it is a right way. Because there are two keys '*' and '?' in normal mode implement the function in vim editor.  Shall we implent it in the same way?
